## PAUP and Parsimony Lab

1. From the iterative alignment section: Which aslignment (pasta_script1, pasta_script2, pasta_script3) do you think is the "best"? By what criteria did you arrive at this decision? 
Given the algorithm for determining homologies (matching bases and not indels) and the fact that these sequence data are random generations I believe that the pasta script2 would be most desirable alignment. This had the greatest number of homologies out of all three granted the third alignment was on a reduced dataset. 
2. In tree searching, when would you expect the swap algorithm to matter strongly?
Swapping algorithms would matter strongly when the number of taxa in the data set is exceedingly large. Using the TBR swapping method would reduce the amount of computational time and would also have the greatest distance between topologies due to it making bigger changes in relation to SPR which would probably be more beneficial for higher order questions with large amounts of taxa.
3. In your own words, what is a bootstrap analysis? Does it make sense to describe a bootstrap value as the probability that a grouping on a tree is correct?
N/A
4. What might be a situation in which you would want to use the jackknife instead of the bootstrap? 
N/A

5. Transfer your trees to your desktop. Use FigTree or IcyTree to view them. Take a screenshot, and save it to the fig folder in the class repository. Below, complete the file path to where you have the file in the fig directory to embed the image of your screen shot. 

![Embed your bootstrap tree here](../fig/your_bootstrap_tree_image) 

![Embed your jackknife tree here](../fig/your_jackknife_tree_image) 


## Add, commit, and push your homework by 5 pm on Friday.

